docs/client/components/common/DefaultCode/Button/index.js

```diff
@@ -4,6 +4,8 @@ class Demo extends React.Component {
     return (
       <div>
         <PreviewBlock header="Styled Buttons">
+        <div style={{display: 'grid',
```

Comment:
Can we add this through CSS

Reply:
I don't think so. Even in the official docs the examples have inline styling. i guess we can create a style object separately so that the code becomes more readable.